specs/vscode_extensions.table

```diff
@@ -0,0 +1,17 @@
+table_name("vscode_extensions")
+description("Lists all vscode extensions.")
+schema([
+    Column("id", TEXT, "Extension id"),
+    Column("version", TEXT, "Extension version"),
+    Column("path", TEXT, "Extension path"),
+    Column("publisher", TEXT, "Publisher Name"),
+    Column("installed_at", BIGINT, "Installed Timestamp"),
+    Column("prerelease", INTEGER, "Pre release version"),
+    Column("uid", BIGINT, "The local user that owns the plugin",
+      index=True),
```

Comment:
This jumped to my eyes, `index=True` not only means that the constraint will be passed to the table for filtering, but also that it identifies a unique row (which means each row must have a different value of `uid`).

I think you want `additional=True` instead for just optimized filtering.